src/evaluator.ts

```diff
@@ -76,9 +80,19 @@ export class Evaluator {
     bucketBy: string,
     percentage: number,
   ): boolean {
-    const property = this.getAttrValue(target, bucketBy);
+    let property = this.getAttrValue(target, bucketBy);
     if (!property) {
-      return false;
+      const old_bucketBy = bucketBy;
+      bucketBy = 'identifier';
+      property = this.getAttrValue(target, bucketBy);
+      if (!property) {
+        return false;
+      }
+      warnBucketByAttributeNotFound(
+        old_bucketBy,
+        property?.toString(),
+        this.log,
+      );
     }
     const bucketId = this.getNormalizedNumber(property, bucketBy);
```

Comment:
If I remember correctly, Node has the property and bucket by backwards in this, can we verify and fix that at the same time?

Reply:
do you mean to keep parameter ordering consistent consistent with other SDKs? yeah should be easy enough